maxcdn.go

```diff
@@ -6,11 +6,12 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/http/httputil"
 	"net/url"
+	"strconv"
 	"strings"
-	"sync"
 
-	"github.com/garyburd/go-oauth/oauth"
+	"github.com/MaxCDN/go-maxcdn/Godeps/_workspace/src/github.com/garyburd/go-oauth/oauth"
```

Comment:
That's a choice in how you use Godeps, I typically use this method, opting not to change the GOPATH. That import line was written directly by Godeps itself. If we use Godeps at all, I greatly prefer this method.